### PR TITLE
[WebProfilerBundle] Fix the values of some CSS properties

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

You can check the valid values of these properties at:

* https://developer.mozilla.org/en-US/docs/Web/CSS/max-width
* https://developer.mozilla.org/en-US/docs/Web/CSS/min-height

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
Thank you @javiereguiluz.

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/exception.css.twig [hunk 1]
```diff
@@ -28,5 +28,5 @@
 }
 
 .exception-message-wrapper .container {
-    min-height: auto;
```
Comment (stof) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/exception.css.twig:31]: `auto` is a valid value (it is even the initial value): https://developer.mozilla.org/en-US/docs/Web/CSS/min-height#values

Comment (stof) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/exception.css.twig:31]: ~~Comparing the "formal specification" section of MDN with https://www.w3.org/TR/css-sizing-3/#sizing-properties, it looks like MDN has the wrong formal spec, as it omits `auto`~~ the "formal syntax" section actually shows `auto`

Comment (javiereguiluz) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/exception.css.twig:31]: OK ... luckily the new value should be equivalent ... but please tell me if I'm wrong. Thanks!

**Pull request merged**